boa/src/syntax/parser/statement/try_stm/catch.rs

```diff
@@ -57,17 +60,63 @@ where
         let catch_param = if cursor.next_if(Punctuator::OpenParen)?.is_some() {
             let catch_param =
                 CatchParameter::new(self.allow_yield, self.allow_await).parse(cursor)?;
+
             cursor.expect(Punctuator::CloseParen, "catch in try statement")?;
             Some(catch_param)
         } else {
             None
         };
 
+        let mut set = HashSet::new();
```

Comment:
We could use `rustc_hash::FxHashSet` here, because we only care about speed here. It should just be an import and type change, as we already use it in other places.

Reply:
Fixed in 0949a41005afaf32571552ac1b257af5acb40ac5!